docs/meetings/team/2023_01_23.md

```diff
@@ -0,0 +1,77 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:00
+
+**End Time** 10:36
+
+**Location** Discord
+
+**Date** January 23, 2023
+
+## 👋 Attendence
+
+1. Veronica
+2. Logan
+3. Edouard
+4. Matt
+
+***Attendence Comments***
+
+- Logan will be away next week for medical reasons
+
+## 🃏 Joke of the Day
+
+What do you call 8 hobbits? A hobbyte
+
+## 📢 Announcements
+
+- Week 3 GitStats due this week on Friday
+- Week 4 has a new deliverable we should ask the prof about
+- Rhubarb developer replied! Check Discord for a screenshot of the message
+
+## 📅 Last Week (Restrospective)
+
+- Thank you, Matt, for submitting GitStats for us!
+- Ed was working on making a formal API definition, but it's incomplete
+- We found that Postman was crashing (screen goes white when trying to preview, but the download works)
+- Veronica implemented blinking, but could implement types to improve it for TypeScript
+
+## 📋 This Week
+
+### Assets
+
+- Official assets with mouths and eyes are exported from Photoshop! Thank you, Logan!
+- If we want to have the arms move...
+  - Need to check with Client if we want breathing vs. poses because poses will require more from the artist
+  - Barb's arms are movable
+  - The boy's arms are attached to the body, which are attached to the feet and are NOT movable
+
+### Issues this week
+
+- Continue working on a formal API definition (Ed)
+- Figure out why Postman is crashing (Ed & Matt)
+  - This could have to do with ffmpeg scaling (Logan)
+  - Matt had it temporarily working after a re-install, but it's broken again
+- Implement more types and clean up current types (Veronica & Ed)
+- Rhubarb developer to-do's:
+  - Extended mouth shapes being used? double check (Veronica)
+  - Check command-line program's exit code rather than its output (Matt & Veronica)
+- Check Boy in Illustrator because his arms might not be movable due to flattening of layers (Veronica)
+- Need to find better test files with harder English words so that we can better showcase what we have (Everyone)
+
+## 🔮 Future Issues (after this week)
+
+- Export the assets in different arm positions after discussing at client meeting
+- If doing poses, will we trigger them randomly? How do we want to do this? Something to think about this week.
+
+## 🤝 Client Meeting
+
+- Showcase videos from both recognizers so that we can ask if they want quality vs. speed for English
+- We should also ask if they want breathing or specific poses (which would require more from the artist)
+- After checking Boy in Illustrator, might need to request asset change
```

Comment:
Does this mean that the arms/legs/body is attached in the illustrator file also?

Reply:
sorry, I didn't write it clearly! I haven't checked yet, I meant, after we check in Illustrator, we might need to... I can check later today/tomorrow!

Reply:
Sounds good!